### PR TITLE
Storybook: CopyText common adapter (using new Toast adapter)

### PR DESCRIPTION
**Pull request by buoyad**
Also more fully mocks out electron (relevant because this uses `copyToClipboard`). The native toast is a little offset to the bottom of the screen but that might have something to do with the storybook environment, I guess it can wait until the screen is implemented. r? @keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/floating-box.desktop.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {Box} from './box'
 import {ModalPositionRelative} from './relative-popup-hoc.desktop'
 import type {Props} from './floating-box'
 
-const StyleOnlyBox = (props: any) => <Box style={props.style} children={props.children} />
+const StyleOnlyBox = (props: any) => <Box children={props.children} />
```
Comment (buoyad) [inline shared/common-adapters/floating-box.desktop.js]: It turns out the style was being applied twice. Not used anywhere until this point.

File: shared/common-adapters/copy-text.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+          </Text>
+        </Toast>
+        <Text
+          type="Body"
+          selectable={true}
+          style={collapseStyles([styles.text, !this.state.revealed && {width: 'auto'}])}
+        >
+          {this.state.revealed ? this.props.text : '••••••••••••'}
+        </Text>
+        {!this.state.revealed && (
```
Comment (adamjspooner) [inline shared/common-adapters/copy-text.js]: Should `withReveal` be here too? My assumption, based on its name, is that one with `withReveal: false` wouldn't have a “Reveal” option at all.

Comment (buoyad) [reply shared/common-adapters/copy-text.js]: Added

File: shared/common-adapters/toast.desktop.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  ...transition('opacity'),
+  '&.visible': {
+    opacity: 1,
+  },
+  opacity: 0,
+})
+
+export default (props: Props) => (
+  <FloatingBox
+    onHidden={() => {}}
```
[the remark this answers is not in the document]
Comment (buoyad) [reply shared/common-adapters/toast.desktop.js]: Yeah that prop could use some better typing, I made it optional.

**Pull request merged**